File: 1375. Bulb Switcher III/1375.c
int numTimesAllBlue(int* light, int lightSize){
    int count = 0;
    int maxNum = 1;

    for (int i=0; i<lightSize; i++){
        if(*(light+i) > maxNum){
            maxNum = *(light+i);
        }

        if((i+1)>=maxNum){
            count++;
        }
    }

    return count;
}

File: 1375. Bulb Switcher III/1375.go
package leetcode

func numTimesAllBlue(light []int) int {
    maxNum := 1
	count := 0

	for i,v := range light{
		if v>maxNum{
			maxNum = v
		}

        if (i+1)>=maxNum{
			count ++
		}
	}

	return count
}